app/views/posts/show.html.erb

```diff
@@ -1,10 +1,21 @@
-<h1>Here is the post number <%= @post[:user_id] %> for the user number <%= @post[:Post_id] %></h1>
-<ul>
-<li>Post: <%= @post[:post_text] %></li>
-<% (1..@post[:comments]).each do |number| %>
-    <li>
-      <h5>Comment from user: <%=number+1 %></h5>
-      <p>This is the commnet text for the user <%= number+1 %></p>
-    </li>
-  <% end %>
-</ul>
\ No newline at end of file
+<div class="container">
+    <div class="card mb-5" style="width: 100%;">
+    </div>
+
+    <div class="card mb-2" style="width: 100%;">
+    <div class="card-body">
+      <h5 class="card-title">Post <%= @post.id %> By <%= @user.name %></h5>
+      <p class="card-text"><%= @post.text %></p>
+      <div class="col-md-3 text-md-end col-sm-3" style="width: 100%;">
+        <p class="text-muted mt-5"><span>Likes: <%= @post.like_counter %></span> <span>Comments: <%= @post.comment_counter %></span></p>
+      </div>
+    </div>
+  </div>
+  <div class="card mb-5" style="width: 100%;">
+  <% @post.comments.each do |comment| %>
+    <div class="card-body">
+      <p class="card-text"><%= comment.user.name %>: <%= comment.text %></p>
+    </div>
+    <% end %> 
+  </div>
```

Comment:
- [ Good job ] It's possible to add a comment to a post using the fields in the student's schema and method arguments shown in the screenshot below.

## Screenshot
![Screenshot from 2023-10-21 15-13-44](https://github.com/mahdinoori2000/blog-app/assets/16392046/1598ae0c-41d0-4061-8a62-5d1ddfba7d51)

## Results
![Screenshot from 2023-10-21 15-16-39](https://github.com/mahdinoori2000/blog-app/assets/16392046/48c21ad4-bf92-455d-9e31-60cde8750023)